extensions/common/gcp/gcp-core/src/main/java/org/eclipse/edc/gcp/iam/IamService.java

```diff
@@ -33,15 +35,9 @@ public interface IamService {
     /**
      * Creates a temporary valid OAunth2.0 access token for the service account
      *
-     * @param serviceAccount The service account the token should be created for
+     * @param serviceAccount service account the token should be created for;
```

Comment:
looking like at the usages of this methods, seems that there's no call with `null` and I think it's fair for an interface not to accept null inputs since we have the `ADC_SERVICE_ACCOUNT` instance.

Reply:
Done